Updates on request and account to be clickable

wmmoua opened the pull request:
Note: Updated the request posts to be able to be clickable. Similar to the browse page when clicking on it.

Also added requests into the account page so it will have posts and requests.
Note: I tested on the Dummy account so quicker to test it out checking the dummy post and request. Trying to look into separating posts and requests in account page for easier access.

wmmoua committed:
Updates on request and account to be clickable

wmmoua committed:
Forgot to add the viewRequest file

neelbhatia1 reviewed (approved):
Functional and bug free

wmmoua committed:
has a message on accont that says account.ejs (Removed)

On the account page there was a text that says account.ejs which was removed in file.
